app/experimenter/nimbus-ui/src/components/SidebarActions/index.tsx

```diff
@@ -30,27 +32,43 @@ export const SidebarActions = ({
       : CHANGELOG_MESSAGES.UNARCHIVING_EXPERIMENT,
   });
 
+  const disabled = !experiment.canArchive || isLoading;
   return (
     <div data-testid={"SidebarActions"}>
-      <p className="edit-divider position-relative small my-2">
+      <div className="edit-divider position-relative small my-2">
         <span className="position-relative bg-light pl-1 pr-2 text-muted">
           Actions
         </span>
-      </p>
-      <p>
+      </div>
+      <div>
         <LinkNav
           useButton
           key="sidebar-actions-archive"
-          disabled={!experiment.canArchive || isLoading}
+          route={`${experiment.slug}/#`}
           testid="action-archive"
           onClick={onUpdateArchived}
+          {...{ disabled }}
         >
           <Trash className="sidebar-icon" />
-          {experiment.isArchived
-            ? "Unarchive Experiment"
-            : "Archive Experiment"}
+          <div>
+            <p className="m-0">
+              {experiment.isArchived ? "Unarchive" : "Archive"}
+              {disabled && (
+                <>
+                  <Info
+                    data-tip="Experiments can only be archived when in Draft or Complete."
```

Comment:
Should this tool tip text be a constant?
Ref: https://github.com/mozilla/experimenter/blob/82c486cc8f3944e27a9ba602f00dbb8c211ae373/app/experimenter/nimbus-ui/src/components/PageEditMetrics/FormMetrics/index.tsx#L147-L154